### PR TITLE
fix the language for simple and proper understanding

### PR DESCRIPTION
**Pull request by kamal-github**
It fixes the language for simple and proper understanding in engines section in Rails guide 

http://guides.rubyonrails.org/engines.html#generating-an-article-resource

### CODE REVIEW

**Comment by fxn** [comment]
Could we simplify further by just saying "inherits"?

**Comment by fxn** [comment]
Could you please also squash all commits and add "[ci skip]" to the message of the single one?

File: guides/source/engines.md [hunk 1]
```diff
@@ -402,7 +402,7 @@ module Blorgh
 end
 ```
 
-NOTE: The `ApplicationController` class being inherited from here is the
+NOTE: The `ArticlesController` class inherits from
 `Blorgh::ApplicationController`, not an application's `ApplicationController`.
```
Comment (maclover7) [inline guides/source/engines.md]: I know this isn't part of your changeset, but can you change the `an` to `the`, here?

**Comment by kamal-github** [comment]
Hey @maclover7, I did the suggested change :)

**Comment by maclover7** [comment]
:+1: Thank you for your contribution!

**Comment by maclover7** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by maclover7** [comment]
:hear_no_evil: Sorry, I forgot to ask you to squash down your commits. When you contribute in the future, try and squash down multiple commits to just one. Thanks! :)

**Comment by kamal-github** [comment]
Thanks!, @maclover7 @fxn :)

**Comment by fxn** [comment]
@maclover7 this was merged without squashing and without message amending, which as asked for above. As a general rule, when someone takes the lead in an issue they are usually the ones that end up merging or rejecting.

File: guides/source/engines.md (hunk 1 above)
Comment (kaspth) [inline guides/source/engines.md]: Engines are not connected to just one application, therefore `an` was correct 😁

**Comment by maclover7** [comment]
:+1: will keep in mind in the future.

**Pull request merged**